WebRoot/public/error.jsp

```diff
@@ -69,37 +69,17 @@
 <c:set value="/img" var="iconPath" scope="request"/>
 <body>
 	<div class="ErrorScreen">
-		<div class="center">
-			<div class="contentBox">
-				<div class="InlineErrorPanel">
-					<table width="100%">
-						<tr>
-							<td width="1%">
-								<img src="<c:url value='/img/dwt/ImgWarning_32.png?v=${version}' />"
-									 title="Error" alt="Error" id="ZErrorIcon">
-							</td>
-							<td><h2 style="margin:0;"><fmt:message key="${errTitle}"/></h2></td>
-						</tr>
-						<tr>
-							<td></td>
-							<td style="border-top:1px solid #333;">
-								<p style="margin:1em 0 2em;"><fmt:message key="${errMsg}"/><br/>
-									<fmt:message key="errorTryAgainLater"/></p>
-								<p style="margin-bottom:2em;">ERROR: ${errCode}</p>
-								<p style="font-size:1.2em;font-weight:bold;margin-bottom:1em;">
-									<a href="/">
-										<span style="font-size:1.5em;">&laquo;</span>
-										<span><fmt:message key="errorGoBack"/></span>
-									</a>
-								</p>
-							</td>
-						</tr>
-					</table>
-				</div>
-			</div>
-			<div class="decor1"></div>
+		<div class="errorBox">
+			<h2><fmt:message key="${errTitle}"/></h2>
+			<p>
+				<fmt:message key="${errMsg}"/><br/>
+				<fmt:message key="errorTryAgainLater"/><br/><br/>
+				Error ${errCode}
```

Comment:
translations?

Reply:
translations are already present , haven’t chnaged the keys have just reskined it

Reply:
`Error ${errCode}` is not translated

Reply:
it is present in same way here in legacy code so i dint add 
<p style="margin-bottom:2em;">ERROR: ${errCode}</p>